build/common.props

```diff
@@ -19,4 +19,7 @@
     <PackageReference Include="Internal.AspNetCore.Sdk" Version="$(InternalAspNetCoreSdkVersion)" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
Nit: in other repos, we're writing this condition as `'$(TargetFrameworkIdentifier)' == '.NETFramework'`